app/views/shared/_email_subscribe_unsubscribe_flash.html.erb

```diff
@@ -5,11 +5,27 @@
 <% if show_email_subscription_success_banner?(@account_flash) %>
   <div class="govuk-grid-row govuk-!-margin-top-3">
     <div class="govuk-grid-column-two-thirds">
-      <%= render "govuk_publishing_components/components/success_alert", {
-        message: email_subscription_success_banner_heading(@account_flash),
-        description: banner_description,
-        margin_bottom: 0,
-      } %>
+      <%
+        ga4_data = {
+          event_name: "form_complete",
+          type: "email subscription",
+          text: email_subscription_success_banner_heading(@account_flash),
```

Comment:
It looks like `email_subscription_success_banner_heading` calls the translation function. It's probably worth ensuring the English translation is used for this text instead.

Reply:
More specifically, it calls the `t()` function - is that different to I81n.t?

Reply:
I think the first one might be an alias for the second, but I'm not 100% certain.

Have altered the method to supply a specific locale if requested, good spot 👍